models/nnunet_prostate_task24/config/default.yml

```diff
@@ -0,0 +1,41 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: default configuration for nnUNet MR Prostate segmentation (dicom to dicom)
+
+execute:
+- DicomImporter
+- NiftiConverter
+- NNUnetRunner
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: 'mr'
```

Comment:
I suggest using `mod: '%Modality'` here to automatically fetch the Modality header from Dicom.